esmvalcore/config-developer.yml

```diff
@@ -26,7 +26,7 @@ CMIP6:
     DKRZ: '{activity}/{institute}/{dataset}/{exp}/{ensemble}/{mip}/{short_name}/{grid}/{latestversion}'
     ETHZ: '{exp}/{mip}/{short_name}/{dataset}/{ensemble}/{grid}/'
   input_file: '{short_name}_{mip}_{dataset}_{exp}_{ensemble}_{grid}*.nc'
-  output_file: '{project}_{dataset}_{mip}_{exp}_{ensemble}_{short_name}_{start_year}-{end_year}'
+  output_file: '{project}_{dataset}_{mip}_{exp}_{ensemble}_{short_name}'
```

Comment:
Maybe add a comment above that `_{start_year}-{end_year}` is appended automatically to `output_file` (except fx variables)?

Reply:
meh

Reply:
I've added the information to the documentation